org.freedesktop.Platform.VulkanLayer.VkCapture.metainfo.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="addon">
+  <id>org.freedesktop.Platform.VulkanLayer.VkCapture</id>
+  <extends>org.freedesktop.Platform</extends>
+  <name>Vulkan/OpenGL Capture tools</name>
```

Comment:
I think the name should be `OBS VkCapture tools`

Reply:
Yeah, I think that makes it clearer that it's supposed to be used with the OBS plugin